deepchecks/nlp/checks/train_test_validation/keyword_frequency_drift.py

```diff
@@ -0,0 +1,142 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module containing the keyword drift check."""
+from deepchecks import CheckResult, ConditionResult, ConditionCategory
+from deepchecks.core import DatasetKind
+from deepchecks.utils.strings import format_number
+from deepchecks.core.errors import DeepchecksValueError
+from deepchecks.utils.distribution.drift import word_counts_drift_plot
+from deepchecks.nlp.base_checks import TrainTestCheck
+from deepchecks.utils.distribution.drift import cramers_v, psi
+from typing import Union, List, Any
+from deepchecks.nlp.context import Context
+from sklearn.feature_extraction.text import TfidfVectorizer
+from nltk.stem.porter import PorterStemmer
+import numpy as np
+
+__all__ = ['KeywordFrequencyDrift']
+
+
+class KeywordFrequencyDrift(TrainTestCheck):
+    """
+    Computes the keywords' frequencies drift between the train and the test datasets.
+
+    Parameters
+    ----------
+    top_n_to_show: int, default: 20
+        How many words will be displayed in the graph.
+    top_n_method: Union[str, List[str]], default: 'top_diff'
+        Decides which method will be used to select the top n words to show. Possible values are:
+        "top_diff" for the top difference between train and test, "top_freq" for the top absolute frequencies, or a list
+        of keywords in which case the words on the list are shown and the "top_n_to_show" parameter is ignored.
+    drift_method: str, default: "cramer_v"
+        Decides which method will be used for drift calculation. Possible values are:
+        "cramer_v" for Cramer's V, "PSI" for Population Stability Index (PSI).
+    """
+
+    def __init__(self,
+                 top_n_to_show: int = 20,
+                 top_n_method: Union[str, List[str]] = 'top_diff',
+                 drift_method='cramer_v',
+                 **kwargs):
+        super().__init__(**kwargs)
+        self.top_n_to_show = top_n_to_show
+        self.top_n_method = top_n_method
+
+        if drift_method == 'PSI':
+            self.drift_method = psi
+        elif drift_method == 'cramer_v':
+            self.drift_method = cramers_v
+        else:
+            raise DeepchecksValueError('drift_method must be one of: PSI, cramer_v')
+        self.stem_func = PorterStemmer().stem
+        self.token_pattern = r'[a-z]{2,}'
+
+        self.top_n_words = None
```

Comment:
Think these two are not used anywhere

Reply:
I'll remove them after setting the final implementation